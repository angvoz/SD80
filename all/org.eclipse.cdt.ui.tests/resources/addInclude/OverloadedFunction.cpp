#include "A.h"

namespace ns3 {

void test() {
  func(0);
}

}