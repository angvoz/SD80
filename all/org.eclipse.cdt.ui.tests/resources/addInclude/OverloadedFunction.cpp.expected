#include "A.h"
#include "OverloadedFunction.h"

using ns3::func;

void test() {
  func(0);
}
